package.json

```diff
@@ -137,7 +137,7 @@
     "audit-ci": "^7.1.0",
     "aws-sdk-client-mock": "^4.1.0",
     "cheerio": "^1.0.0",
-    "chokidar": "^4.0.3",
+    "chokidar": "^3.6.0",
```

Comment:
Not strictly related to this PR, but the update of this dep (by renovate) was incompatible with nunjucks and prevents nunjucks doing live-reloads when in dev mode
I've set it back to the original version, and added an exclusion in the renovate config (which is what other DPS services appear to be doing for the same reason)